### PR TITLE
Add example to `optuna.study.Study.get_trials` 

### PR DESCRIPTION
**Pull request by norihitoishida**
## Motivation
This PR is related to #1334

## Description of the changes
Update the docstring of `Study.get_trials()`

### CODE REVIEW

File: optuna/study.py [hunk 1]
```diff
@@ -114,6 +114,20 @@ def get_trials(self, deepcopy: bool = True) -> List[FrozenTrial]:
...
+
+                def objective(trial):
+                    x = trial.suggest_uniform("x", -1, 1)
+                    return x ** 2
+
+                study = optuna.create_study()
+                study.optimize(objective, n_trials=3)
+
+                trials = study.get_trials()
+
```
Comment (toshihikoyanase) [inline optuna/study.py:130]: How about adding an assertion like this? It shows that `get_trials` returns all trial clearly.
```suggestion
                trials = study.get_trials()
                assert len(trials) == 3
```

**Review by toshihikoyanase — COMMENTED** [review]
Thank you for your PRs as always. It basically looks good to me, but I added a suggestion.

File: optuna/study.py (hunk 1 above)
Comment (norihitoishida) [reply optuna/study.py]: Thanks a lot!

**Review by hvy — APPROVED** [review]
LGTM!

**Pull request merged**